### PR TITLE
Fixes  test and review ACC-143

### PR DESCRIPTION
**Pull request by MarcinR**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/ACC-143" title="ACC-143" target="_blank">StoryACC-143</a>  Context change on focus (3.2.1)
  </td></table>
  <br />
 

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Calling/CallInfoViewController/CallStatusView.swift [hunk 1]
```diff
@@ -29,7 +29,6 @@ enum CallStatusViewState: Equatable {
     case terminating // "Ending call..."
 }
 
-
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Calling/CallInfoViewController/CallStatusView.swift:32]: we don't need these changes

File: Wire-iOS/Sources/UserInterface/Calling/CallGridView/CallParticipantViews/BaseCallParticipantView.swift [hunk 2]
```diff
@@ -174,13 +194,14 @@ class BaseCallParticipantView: OrientableView, AVSIdentifierProvider {
...
             avatarView.widthAnchor.constraint(equalToConstant: 72),
-            avatarView.widthAnchor.constraint(lessThanOrEqualTo: widthAnchor, multiplier: 0.7),
+            avatarView.widthAnchor.constraint(lessThanOrEqualTo: widthAnchor, multiplier: 0.7).withPriority(.required),
             avatarView.heightAnchor.constraint(equalTo: avatarView.widthAnchor).withPriority(.required),
             userDetailsView.topAnchor.constraint(greaterThanOrEqualTo: avatarView.bottomAnchor, constant: 16).withPriority(.defaultHigh - 1),
             avatarView.topAnchor.constraint(greaterThanOrEqualTo: topAnchor).withPriority(.required)
         ])
     }
 
+
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Calling/CallGridView/CallParticipantViews/BaseCallParticipantView.swift:204]: there are few swift lint issues

File: Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/BottomSheetContainerViewController.swift [hunk 3]
```diff
@@ -129,7 +129,7 @@ class BottomSheetContainerViewController: UIViewController {
 
     }
 
-    public func didChangeState() {}
+    public func didChangeState() {} //for overriding
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/BottomSheetContainerViewController.swift:132]: some comments

Comment (MarcinR) [reply Wire-iOS/Sources/UserInterface/Calling/BottomSheetContainerViewController/BottomSheetContainerViewController.swift:132]: yes left it so no one delete it thinking empty method is doing nothin

File: Wire-iOS/Sources/UserInterface/Calling/CallInfoViewController/CallStatusView.swift (hunk 1 above)
Comment (MarcinR) [reply Wire-iOS/Sources/UserInterface/Calling/CallInfoViewController/CallStatusView.swift:32]: I don't get it. That's just removed empty line

File: Wire-iOS/Sources/UserInterface/Calling/CallGridView/CallParticipantViews/BaseCallParticipantView.swift (hunk 2 above)
Comment (MarcinR) [reply Wire-iOS/Sources/UserInterface/Calling/CallGridView/CallParticipantViews/BaseCallParticipantView.swift:204]: done

File: Wire-iOS/Sources/UserInterface/Calling/CallInfoViewController/CallStatusView.swift (hunk 1 above)
Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/Calling/CallInfoViewController/CallStatusView.swift:32]: But why?

**Review by KaterinaWire — APPROVED** [review]

**Pull request merged**